frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/storage-tab.tsx

```diff
@@ -89,6 +89,13 @@ const StorageTabFirehose: React.FC<StorageTabFirehoseProps> = ({
   isDeleteDisabled,
 }) => {
   const { t } = useTranslation();
+
+  React.useEffect(() => {
+    storages.filter(
+      (storage) => storage.type === VMWizardStorageType.TEMPLATE && removeStorage(storage.id),
```

Comment:
do we want to remove **all** disks from a template ? can we remove  only the ones that have bad names (e.g. ${name}) ?